server/src/test/java/org/apache/iotdb/db/metadata/MTreeTest.java

```diff
@@ -310,113 +310,9 @@ public void testAddAndQueryPathWithAlias() {
     }
   }
 
-  @Test
```

Comment:
Why we remove this test?

Reply:
This test doesn't reflect any current application or usage of mtree, thus it is removed.